README.md

```diff
@@ -62,6 +62,33 @@ hzc --help
 # Non-interactive mode
 hzc map --name myMap put --key myKey --value myValue
 ```
+### Keyboard Shortcuts
+Hazelcast CLC inherits following shortcuts from [c-bata/go-prompt](https://github.com/c-bata/go-prompt)
+
+Emacs-like keyboard shortcuts are available by default (these also are the default shortcuts in Bash shell).
+
+Key Binding          | Description
+---------------------|---------------------------------------------------------
+<kbd>Ctrl + A</kbd>  | Go to the beginning of the line (Home)
+<kbd>Ctrl + E</kbd>  | Go to the end of the line (End)
+<kbd>Ctrl + P</kbd>  | Previous command (Up arrow)
+<kbd>Ctrl + N</kbd>  | Next command (Down arrow)
+<kbd>Ctrl + F</kbd>  | Forward one character
+<kbd>Ctrl + B</kbd>  | Backward one character
+<kbd>Ctrl + D</kbd>  | Delete character under the cursor
+<kbd>Ctrl + H</kbd>  | Delete character before the cursor (Backspace)
+<kbd>Ctrl + W</kbd>  | Cut the word before the cursor to the clipboard
+<kbd>Ctrl + K</kbd>  | Cut the line after the cursor to the clipboard
+<kbd>Ctrl + U</kbd>  | Cut the line before the cursor to the clipboard
+<kbd>Ctrl + L</kbd>  | Clear the screen
+With few additions:
```

Comment:
How about adding a blank line before this one?

Reply:
Fixed at https://github.com/hazelcast/hazelcast-commandline-client/pull/44/commits/9c8793bb9e42a52b4ba2005d574530a43b8d9a69